README.md

```diff
@@ -1 +1,131 @@
-# rfq-marketplace
\ No newline at end of file
+# rfq-marketplace
+
+rfq-marketplace is a contract that provides a token exchange platform.
+
+## Prerequisites
+
+- Node.js
+- NPM
+
+## Deploy the contract
+
+The first step, clone the project and install the dependency.
+```
+git clone https://github.com/second-state/rfq-marketplace.git
+cd rfq-marketplace
+npm install
+```
+You can edit the `hardhat.config.js` to choose which network and account to deploy the contract.
+In the example, we use CyberMiles chain to demo.
+
+Next step, run the following command to run the deploy script.
+```
+npx hardhat run script/deploy.js --network cyber
```

Comment:
This should be `scripts` right?

I would also like to use `cybermiles` instead of `cyber` here.